libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -825,7 +825,11 @@ void NavEKF2_core::writeExtNavData(const Vector3f &sensOffset, const Vector3f &p
 
     extNavDataNew.pos = pos;
     extNavDataNew.quat = quat;
-    extNavDataNew.posErr = posErr;
+    if (posErr == 0) {
```

Comment:
is_zero(posErr)

Reply:
Yes, thank you
travis-ci also failed with "comparing floating point with == or != is unsafe"